packages/adp-tooling/src/types.ts

```diff
@@ -39,3 +39,52 @@ export interface AdpWriterConfig {
         description?: string;
     };
 }
+
+export interface ManifestAppdescr {
+    fileName: string;
+    layer: string;
+    fileType: string;
+    reference: string;
+    id: string;
+    namespace: string;
+    version: string;
+    content: Content[];
+}
+
+export interface Content {
+    changeType: string;
+    content: object;
+    texts?: object;
+}
+
+export const enum FolderNames {
```

Comment:
Are these folder really static? I know that by default an adaptation project uses these folders but technically, I can put in my change anything else and it will also work, correct? So, please think about how to not use them.

Reply:
Yes, they are static and are predefined for adaptation projects. The structure is always the same. Maybe I didn't understand about you putting your 'change anything else' but you cannot put .change files anywhere else than the changes folder (not in any subfolder)

Reply:
`changes` is the only one fixed (at least I haven't tried to change this one). `webapp` can be anything else, it is only relevant in your development projects to define the source root that will be used to build to `dist`. I can put my fragments in any folder under changes, only the reference fro the `.change` file matters. The same for i18n files, controller extensions etc.
So, don't use these hard coded folder names but you have to e.g. find your fragments using the project API (to not rely on `webapp` and then globs like `**/*.fragment.xml`.

Reply:
You are only using `Changes` and `Fragments` in your code. I understand that `Coding` and `Internationalization` will be used later, however, I would remove them for now and only add them when you start using them (in future PRs)